app/models/jobsModel.js

```diff
@@ -54,8 +55,28 @@ async function get(filter){
     }
 }
 
+async function dequeue(){
+    try{
+        const presentDate = new Date();
+        presentDate.setMinutes(presentDate.getMinutes() + 20);
+        const toBeSentJobs = await db('jobs')
```

Comment:
Separating the knex function call from the dequeue function might be a good idea if we wanted to change the start date or if we wanted to dequeue based on class id, etc.

Reply:
Good comment, but this function is a knex model function, so it can stay as it is. and the function Jacob is working on returns all jobs that is going to happen with in the next 20-30 mins (this interval can be flexible using parameters), but we are not dequeuing based on class id.